transformer_engine/pytorch/module/base.py

```diff
@@ -877,3 +877,22 @@ def get_fp8_weights_scratchpad(
         is_first_microbatch: Union[bool, None],
     ) -> List[torch.Tensor]:
         """Needs override."""
+
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                            missing_keys, unexpected_keys, error_msgs):
+        """
+            The function loads an extra state containing fp8_meta weights. 
```

Comment:
Is this function specifically testing the `primary_weights_in_fp8` method and it doesn't have any effect in other scenarios? If yes, we should mention it and make it clear. 
Nits:
1. fp8_meta "tensors" not "weights"
2. copy_ not _copy
3. bf16 or fp16 (or even fp32 in fact) and not just fp16

Reply:
Fixed